junit/junit.bzl

```diff
@@ -31,3 +31,5 @@ def junit_repositories(maven_servers = _default_maven_server_urls()):
         name = "io_bazel_rules_scala/dependency/hamcrest/hamcrest_core",
         actual = "@io_bazel_rules_scala_org_hamcrest_hamcrest_core//jar",
     )
+
+    native.register_toolchains("@io_bazel_rules_scala//testing:testing_toolchain")
```

Comment:
This is the default? For people who don't override?

Reply:
Yes, but probably it will stay here only until Specs2 and ScalaTest will be migrated. Each test lib will have it's own configuration.